### PR TITLE
Translate strings to Spanish

### PR DESCRIPTION
**Pull request by chrismytton**

### CODE REVIEW

File: _includes/person_profile.html [hunk 1]
```diff
@@ -45,45 +45,45 @@
...
     {% if person.birth_date %}
-    <dt>Date of birth</dt>
+    <dt>Fecha de nacimiento</dt>
     <dd>{{ person.birth_date }}</dd>
     {% endif %}
 
   </dl>
 
-  <h2>Photo Credit:</h2>
+  <h2>Crédito de la foto:</h2>
```
Comment (JenMysoc) [inline _includes/person_profile.html:81]: Foto cortesia de: sounds better to me though both are understandable

Comment (chrismytton) [inline _includes/person_profile.html:81]: I got this translation from the [YNR translations file for Costa Rica](https://github.com/mysociety/yournextrepresentative/blob/e515df71f76a4a153798208e5321119e3312a34c/locale/es_CR/LC_MESSAGES/django.po#L1782), so I think this should be OK.

**Comment by chrismytton** [comment]
Thanks for the review @JenMysoc!

File: index.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
   <h1 class="page-heading">{{ site.title }}</h1>
 
   <select class="js-canton-selector canton-selector">
-    <option>Select your canton</option>
+    <option>Ingresá tu cantón</option>
```
Comment (JenMysoc) [inline index.html:10]: This needs to read "Elige tu cantón" Ingresar means "admit" or similar...like admit to a hospital

**Pull request merged**